Assets/Scripts/Models/Inventory.cs

```diff
@@ -191,4 +159,13 @@ private void ImportPrototypeSettings(int defaulMaxStackSize, float defaultBasePr
             Category = defaultCategory;
         }
     }
+
+    private void OnStackSizeChanged(Inventory inventory)
```

Comment:
Would calling this something like TriggerStackSizeChanged make more sense, since in a way you want to send a StackSizeChanged event, and not do something when the StackSizeChanged. I might be wrong though.